files/docker/db-sync/guildsetup/cfg/env

```diff
@@ -0,0 +1,119 @@
+## VARIABLES ##
+CURL_TIMEOUT=60
+PGDATABASE=cexplorer
+BRANCH=alpha
+URL_RAW="https://raw.githubusercontent.com/cardano-community/guild-operators/${BRANCH}"
+DB_SCRIPTS_URL="${URL_RAW}/scripts/grest-helper-scripts/db-scripts"
+
+
+## FUNCTIONS ##
+
+
+# Description : Check and apply delta updates using different combinations to retain custom config
+#             : $1 = name of script to update
+#             : $2 = [Y|N] ignore question and auto update
+#             : $3 = [Y|N] do a complete file comparision
+#             : $4 = [Y|N] wait to acknowledge update
+#             : $5 = optional alternative scripts folder instead of default 'cnode-helper-scripts'
+# return code : 0 = no update
+#             : 1 = update applied
+#             : 2 = update failed
+checkUpdate() {
```

Comment:
Just a note , since we hit something like this before - duplicating functions, especially when using same name - can mean if in weeks/months in  future the two env are sourced and not the same, the results may not be expected - can often lead to hiccups, not fully sure I understood the reason to duplicate the file, but am sure you guys would've discussed it. If needed would it make sense to rename the function to distinguish?

Reply:
For sure, I also don't like it, and had a comment there to explain. For now, I just duplicated the part of the `env` that the `setup-grest.sh` script needs to execute the query deployments.